include/thingset.h

```diff
@@ -1358,6 +1358,8 @@ enum thingset_type
     THINGSET_TYPE_SUBSET,  /**< Subset of data items */
     THINGSET_TYPE_FN_VOID, /**< Function with void return value */
     THINGSET_TYPE_FN_I32,  /**< Function with int32_t return value */
+    /**< Element of an array (used when transmitting over classical CAN) */
+    THINGSET_TYPE_ARRAY_ELEMENT,
```

Comment:
The `<` is only used when the comment is *behind* the item we are documenting.

```suggestion
    /** Element of an array (used when transmitting over classical CAN) */
    THINGSET_TYPE_ARRAY_ELEMENT,
```